### DISCUSSION TITLE
How could I support/ contribute?

### DISCUSSION THREAD

**Discussion by joesmart90** [Nachtzuster/BirdNET-Pi#220] [General]
Hi! I'm new here, but I got my pi5 set up and running and have been tracking on bird weather :) 

I have a week off work next week, and was hoping I could spend some time on a project. Maybe even a contribution. I wondered if you might have some ideas about what I could investigate?

I'm a data scientist and have reasonable python, good at pandas and can use some of the charting tools, but haven't done many big code projects. So this would be a first for me. Thanks in advance for any ideas :)

**Comment by Emmo213** [comment]
Good luck. I stopped contributing because even if you fix actual bugs there's a lot of unnecessary gatekeeping with this project. Your best bet would be to create a fork and make any changes you want in your own project.

**Comment by Nachtzuster** [comment]
@joesmart90 There isn't really a to-do list here, since the project is pretty mature, I guess. Though there are plenty of crufty corners still. There always the issues page of course and some Python depencencies might benefit from an update.
